pkg/controllers/podgroup_controller.go

```diff
@@ -99,7 +99,20 @@ func (r *PodGroupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		log.Error(err, "List pods for group failed")
 		return ctrl.Result{}, err
 	}
-	pods := podList.Items
+	pods := podList.DeepCopy().Items
+	// also list pods by the deprecated key for compatibility.
```

Comment:
we don't need to actually, given the version is alpha.

Also I'm planning to standardize the group name in this release. So it's a breaking change anyway.

Reply:
@denkensk WDYT?